plugins/ua_log_stdout.c

```diff
@@ -48,8 +27,14 @@ const char *logCategoryNames[7] = {"network", "channel", "session", "server",
 __attribute__((__format__(__printf__, 4 , 0)))
 #endif
 void
-UA_Log_Stdout_log(void *_, UA_LogLevel level, UA_LogCategory category,
+UA_Log_Stdout_log(void *context, UA_LogLevel level, UA_LogCategory category,
                   const char *msg, va_list args) {
+
+    /* Assume that context is casted to UA_LogLevel */
+    /* TODO we may later change this to a struct with bitfields to filter on category */
+    if ( context != NULL && (UA_LogLevel)context > level )
```

Comment:
You might have to cast to (uintptr_t) first.
Some compilers warn for a direct cast to enums.